app/models/concerns/hyku_addons/solr_document_behavior.rb

```diff
@@ -94,5 +94,12 @@ def attributes
         subject: 'keyword_tesim'
       )
     end
+
+    # Work out the reader class from the solr document model
+    def meta_reader_class
+      "Bolognese::Readers::#{@model.instance_variable_get(:@model)}Reader".constantize
+    rescue NameError
+      "Bolognese::Readers::GenericWorkReader".constantize
```

Comment:
Can we simply return a `Bolognese::Readers::GenericWorkReader` here?

Reply:
Unfortunately not, we need to be able to specify unique fields within the readers that match their Work Form.

Reply:
I get that for line 100 but couldn't line 102 be simplified?
```suggestion
      Bolognese::Readers::GenericWorkReader
```

Reply:
Yea, in hindsight, that seems like the obvious solution.